server/controllers/scripts.js

```diff
@@ -0,0 +1,136 @@
+import knexModule from 'knex';
+import bookshelfModule from 'bookshelf';
+import scriptsService from '../db/services/scripts';
+import { development as devconfig } from '../../knexfile';
+import Script from '../db/models/scripts';
+// import bookshelfCreateTable from '../db/init';
+
+const knex = knexModule(devconfig);
+const bookshelf = bookshelfModule(knex);
+const ScriptModel = Script(bookshelf);
+
+export function saveNewScript(req, res, next) {
+  console.log(req.body, 'params in saveNewScript');
+  const params = {
+    name: req.body.name,
+    body: req.body.body,
+    description: req.body.description
+  };
+
+  scriptsService.saveNewScript(params, ScriptModel)
+    .then((script) => {
+      if (script) {
+        res.status(201).json(script);
+      } else {
+        next();
+      }
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(401).json({ message: 'Could not save script.' });
```

Comment:
The default behavior is to return a 500 error, usually the console error trace formatted as an HTML page.